companies/digitalo.json

```diff
@@ -0,0 +1,25 @@
+{
+    "slug": "digitalo",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Re-In Retail International GmbH",
+    "runs": [
+        "Re-In Retail International GmbH",
+        "Digitalo",
+        "digitalo.de"
+    ],
+    "address": "Nordring 98a\n90409 Nürnberg\nDeutschland",
+    "phone": "+49 911 27430933",
+    "fax": "+49 911 27430934",
+    "email": "datenschutz@digitalo.de",
+    "web": "https://www.digitalo.de/",
+    "sources": [
+        "https://www.digitalo.de/Impressum.html",
+        "https://www.digitalo.de/Datenschutz.html"
+        "https://github.com/datenanfragen/data/pull/1299#issuecomment-926918168"
+    ],
+    "request-language": "de",
```

Comment:
I don't think this record meets our [criteria for setting the `request-language`](https://github.com/datenanfragen/data#request-language), you can remove that.